client/src/plus/ai-help/index.tsx

```diff
@@ -257,12 +257,12 @@ export function AIHelpInner() {
                               </div>
                             );
                           },
-                          code: ({ inline, className, children, ...props }) => {
+                          code: ({ className, children, ...props }) => {
```

Comment:
I notice `<code>` tags get an ugly `node="[object Object]"` attribute set on them, due to `...props`, we should destructure it to remove it:
```suggestion
                          code: ({ className, children, node, ...props }) => {
```

Reply:
I'll open a separate PR for it.